services/zenoss/config.go

```diff
@@ -71,6 +71,8 @@ type Config struct {
 	Type string `toml:"type" override:"type"`
 	// Event TID.
 	TID int64 `toml:"tid" override:"tid"`
+	// Collector name.
```

Comment:
Pls remove ServiceNow of `Username` and `Password` property comments

Reply:
Done, thanks.